### PR TITLE
Update README.md task 2

### PR DESCRIPTION
**Pull request by jalas167**

### CODE REVIEW

File: tasks/2-code-sandbox/README.md [hunk 1]
```diff
@@ -60,9 +60,9 @@ In this task, you will create a web page that will serve as a code sandbox for a
 
 ### Useful links to solve this task
 
-- https://docs.golem.network/creators/javascript/high-level/task-model/
-- https://docs.golem.network/creators/javascript/examples/hello/
-- https://docs.golem.network/creators/javascript/examples/web/
+- https://docs.golem.network/docs/creators/javascript/high-level/task-model/
```
Comment (cryptobench) [inline tasks/2-code-sandbox/README.md:63]: Broken link

File: tasks/2-code-sandbox/README.md [hunk 2]
```diff
@@ -60,9 +60,9 @@ In this task, you will create a web page that will serve as a code sandbox for a
...
+- https://docs.golem.network/docs/creators/javascript/examples/hello/
```
Comment (cryptobench) [inline tasks/2-code-sandbox/README.md:64]: Broken link

File: tasks/2-code-sandbox/README.md [hunk 3]
```diff
@@ -60,9 +60,9 @@ In this task, you will create a web page that will serve as a code sandbox for a
...
+- https://docs.golem.network/docs/creators/javascript/examples/web/
```
Comment (cryptobench) [inline tasks/2-code-sandbox/README.md:65]: Broken link

File: tasks/2-code-sandbox/README.md [hunk 4]
```diff
@@ -103,8 +103,8 @@ can be found at **https://preview.golem.network/**
 
 ## Useful links:
 
-- Javascript docs: https://docs.golem.network/creators/javascript/quickstart/
-- Dapps docs: https://docs.golem.network/creators/dapps/ 
+- Javascript docs: https://docs.golem.network/docs/creators/javascript/quickstart/
```
Comment (cryptobench) [inline tasks/2-code-sandbox/README.md:106]: Broken link, should be https://docs.golem.network/docs/creators/javascript/quickstarts/quickstart

**Review by cryptobench — CHANGES_REQUESTED** [review]

**Comment by jalas167** [comment]
will not patch, as mid-level api docs are no longer available, and removing the links would remove the context

**Pull request closed without merging**